app/Http/Controllers/AccountController.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Account;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+
+class AccountController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+        $id = auth()->user()->id;
+        $income_sum = DB::table('accounts')->where('user_id', '=', $id)->where('income', true)->sum('amount'); //int
```

Comment:
You can use the relations:
$user = auth()->user();
$income_sum = $user->accounts->where('income', true)->sum('amount');